contracts/lib/MultiTokenTransferLib.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.11;
+
+import "./CurrencyTransferLib.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC721/IERC721Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC1155/IERC1155Upgradeable.sol";
+
+library MultiTokenTransferLib {
```

Comment:
let's keep library functions visibility to `internal`
library with public function is a little annoying to deal with cause requires linking and what not. most of the time it's not really necessary.

Reply:
should this be `BundleTransferLib`?
maybe pick 1 `MultiToken` or `Bundle`? since they feel like referring to the same thing.

Reply:
Sticking to `MultiToken` instead of `Bundle` terminology, then